app/main/posts/modify/post-value-edit.html

```diff
@@ -1,229 +1,286 @@
+                'required': attribute.required
+            }">{{attribute.label}}</label>
+            <!-- Attribute Instructions -->
+            <p>{{attribute.instructions}}</p>
 
+            <!-- attributes which use the form-field structure -->
+            <div ng-switch="attribute.input">
+                <!-- type: radio -->
+                <div ng-switch-when="radio">
+                    <div ng-repeat="(key, value) in post.values[attribute.key] track by key">
+                    <div class="form-field radio" ng-repeat="option in attribute.options">
+                        <label>
+                            <input
+                                name="values_{{attribute.id}}"
+                                type="radio"
+                                ng-model="post.values[attribute.key][key]"
+                                ng-required="attribute.required"
+                                value="{{option}}"
+                            >
+                            {{option}}
+                        </label>
+                    </div>
+                    </div>
+                </div>
+                <!-- type: checkbox -->
+                <div ng-switch-when="checkbox">
+                    <div class="form-field checkbox" ng-repeat="option in attribute.options">
+                    <label>
+                        <input
+                            type="checkbox"
+                            checklist-model="post.values[attribute.key]"
+                            ng-required="attribute.required"
+                            name="values_{{attribute.id}}_{{option}}"
+                            checklist-value="option"
+                            value="option"
+                        >
+                            {{option}}
+                    </label>
+                    </div>
+                </div>
+                <!-- type: categories -->
+                <div ng-switch-when="tags">
+                    <category-selector
+                        available="attribute.options"
+                        selected="post.values[attribute.key]",
+                        form="form"
+                    ></category-selector>
+                    <add-category
+                        form-id="post.form.id"
+                        attribute="attribute"
+                        post-value="post.values[attribute.key]"
+                    ></add-category>
+                </div>
+                <div class="alert error" ng-show="form['values_' + attribute.id].$dirty && taskIsMarkedCompleted()" ng-repeat="(error, value) in form['values_' + attribute.id].$error">
+                    <svg class="iconic">
+                    <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="/img/iconic-sprite.svg#warning"></use>
+                    </svg>
+                    <span translate="{{'post.valid.values.' + error}}" translate-values="{ label : attribute.label }"></span>
+                </div>
```

Comment:
@willdoran not sure if from this PR, but there's a </div> missing here before </fieldset>

Reply:
Think that's fixed now